source/configure/user-management-configuration-settings.rst

```diff
@@ -105,11 +105,24 @@ To delete a user from your Mattermost deployment, you can deactivate the user's
 .. image:: ../images/deactivate-user.png
   :alt: Deactivate a user in Mattermost using the System Console.
 
-5. If the user is deactivated, select **Activate** to re-activate the user.
+You can re-activate a deactivated user by selecting **Activate**.
 
 .. image:: ../images/activate-user.png
   :alt: Activate a user in Mattermost using the System Console.
 
+What happens to deactivated user integrations?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. important::
+
+  If you deactivate a Mattermost user who has integrations tied to their user account, consider the following consequences and recommendations based on the integration type:
+
+  - **Slash commands** will continue to work after user deactivation. You may want to consider deleting the existing slash command token and creating a new slash command associated with a different user account to decouple sensitive token data from the deactivated user account.
```

Comment:
We can also suggest that they regenerate the token and that they check that they check that the deactivated user doesn't have access to the slash command `Request URL` as this is the callback URL to receive the HTTP POST or GET event request when the slash command is run.